app/services/singInService.js

```diff
@@ -1 +1,23 @@
-// # note: nothing really works here
+const user = require('./../models/user'),
+  logger = require('./../logger'),
+  singin = require('./../services/singInService');
+
+exports.singup = (req, resp, err) => {
+  singin.validateReq(req);
+  if (err) logger.log(`error saving your data ${err}`);
+  else {
+    const newuser = user.build({
```

Comment:
why are you creating a user here?

Reply:
we . might ignore this, as in the last PR a refactor was done, so this file does not exist anymore.
you are seeing this because this branch is a few commits before master.